### ISSUE TITLE
Accessibility review

### ISSUE THREAD

**Issue by Sorixelle** [pebble-dev/developer.rebble.io#35]
Some parts of the site (eg. colour contrast) aren't up to scratch with WCAG guidelines. We should look out for areas that need improvement, and address them so we can keep the site easy to use for everyone.

Known issues:
- [ ] White text on `alert--bg-orange` fails AA for contrast
